src/instructlab/lab.py

```diff
@@ -77,8 +77,8 @@ def format_epilog(self, ctx, formatter):
     "--config",
     "config_file",
     type=click.Path(),
-    default=lambda: cfg.DEFAULTS.CONFIG_FILE,
-    show_default="config in user config directory",
+    default=cfg.DEFAULTS.CONFIG_FILE,
```

Comment:
Does `--help` not call the default callback?

Reply:
self answer: no, it does not. `--help` displays `(dynamic)` for options with callbacks. We cannot use `ctx.default_map` either, because the `ilab` callback hasn't been executed, yet.

Reply:
That does not seem to be the case.